process/block/shardblock_test.go

```diff
@@ -1234,6 +1234,580 @@ func TestShardProcessor_ProcessBlockCrossShardWithMetaShouldPass(t *testing.T) {
 	assert.False(t, wasCalled)
 }
 
+func TestShardProcessor_ProcessBlockHaveTimeLessThanZeroShouldErr(t *testing.T) {
+	t.Parallel()
```

Comment:
There are lots of code sequences that are repeating in many tests. 
For example creating the block header (with the miniblock hashes), creating the block body (with all transactions added in the pool holder), these can be extracted in helper function/s and just call those instead of duplicating the code.

Reply:
done